subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/graph/builder/SelectorState.java

```diff
@@ -75,6 +78,11 @@
     private boolean softForced;
     private boolean fromLock;
 
+    // The following state needs to be tracked to consistently construct `` independent of the order dependencies are vistited
```

Comment:
Looks like there are words missing in this comment

Reply:
Woops. Forgot to finish that. Thanks!